### ISSUE TITLE
Documentation is wrong about how to call this...

### ISSUE THREAD

**Issue by mattslay** [s0ph1e/node-website-scraper#28]
I had to use this code in my index.js to run scraper from the command in a terminal window:

var scraper = require('website-scraper');
scraper.scrape(options);

But your samples on the front page show this:

var scrape = require('website-scraper');
scrape(options);

So, I had to use it like an object, but your code shows using it like a function.  I saw where there was a previous issue that is now closed that discussed this, but I pulled my hair out trying to call it like a function according to your docs.  When I used the object syntax, it worked fine.

What am I missing here?

Using version: 0.3.3

**Comment by s0ph1e** [comment]
Hi @mattslay 

I'm sorry that you had such problems with documentation

Now `master` branch contains code and documentation for next release - [1.0.0](https://github.com/s0ph1e/node-website-scraper/milestones/v1.0.0) which is in development

Last published version is 0.3.3. 
You can find documentation for it on [npm page](https://www.npmjs.com/package/website-scraper) or on github in appropriate [release 0.3.3](https://github.com/s0ph1e/node-website-scraper/tree/v0.3.3)

**Comment by mattslay** [comment]
Oh, I see. I'm pretty new at this node/npm stuff, so I am still learning...

I did not understand this, but now I do. Thanks.  I should have been reading documentation from the npmjs.com site for the current released version, instead of coming to this github repo and reading on current master. I did not realize that Master has not not been set as a formal release yet.

I will understand this in the future with npm packages.
